test/TESTING_GUIDELINES.md

```diff
@@ -25,6 +44,17 @@ Our UI tests also run on BrowserStack. If you would like to run them on your Bro
   - Run the test server image with tag `onfido-web-sdk:ui-mock-server` and wait until it responds.
   - Run the test scenarios against test server at `https://localhost:8080`.
 
+#### Running against different browsers locally
+
+- Within `config.json` you simply need to modify the `browserName` value to any of the ones illustrated
+
+`"browsers": [ { "browserName": "chrome | firefox | safari | MicrosoftEdge", "remote": false }`
+
+### BrowserStack notes
+
+- Specific to Safari on BrowserStack, you need to explicitly set the `localhostUrl` value in `config.json` to
+  be read as `"localhostUrl": "https://bs-local.com:8080/",` this is due to the way Safari handles `https://localhost`.
+
```

Comment:
I think it would be worth including these configuration in `config.json` but only execute them if we pass flag ie. `npm run test:ui --browser safari`

Reply:
Well the thing is, the idea here is not to execute based on Browser, but config file. This is the next step because I didn't want to create all the config files with different settings without us having an agreed upon approach.

I can certainly implement the use of an environment variable, which would be my preference because if i pass in `--browser safari` ... are we implying local safari or BS safari? Also, would we choose a default browser if one is not specified? I'm just a little hesitant to add something like this now. My preference would be a specific config file then have the browser you want to run within that as a parameter.

Reply:
I see. I agree, this can be handled as part of another story

Reply:
Great. I think we already have a ticket for this in the backlog.